src/Auth0ResourceOwner.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace Riskio\OAuth2\Client\Provider;
+
+use League\OAuth2\Client\Provider\ResourceOwnerInterface;
+use League\OAuth2\Client\Tool\ArrayAccessorTrait;
+
+class Auth0ResourceOwner implements ResourceOwnerInterface
+{
+    use ArrayAccessorTrait;
+
+    /**
+     * @var array
+     */
+    protected $response;
+
+    public function __construct(array $response = [])
+    {
+        $this->response = $response;
+    }
+
+    public function getId()
+    {
+        return $this->getValueByKey($this->response, 'user_id');
+    }
+
+    public function getEmail()
+    {
+        return $this->getValueByKey($this->response, 'email');
+    }
+
+    public function getName()
+    {
+        return $this->getValueByKey($this->response, 'name');
+    }
+
+    public function getNickname()
+    {
+        return $this->getValueByKey($this->response, 'nickname');
+    }
+
+    public function getIdentities()
```

Comment:
I provided phpdoc comments for public methods in this class. Your implementation does not have that. Is that deliberate?

Reply:
Overall, the changes are excellent.